### DISCUSSION TITLE
Welcome to sv06 Discussions!

### DISCUSSION THREAD

**Discussion by hillsoftware** [hillsoftware/sv06#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Milkman0001** [comment]
I am going to install the UBL firmware and wanted to know about the filament sensor support. Where or what port do I plug the filament sensor into when running this firmware?

**Comment by JabroNole** [comment]
I think there is an issue with using linear advance and s-curve acceleration at the same time.  I would disable s curve and see how that works.  I tried to compile with those options, but ran into errors in VSC.

**Comment by mig82au** [comment]
I've tried to build to do exactly that but I have no idea how to compile
Marlin. At high speed with a curve, any linear advance causes terrible gaps
in the print. Setting LA to 0 greatly improved very fast prints.
